wsd/AdminModel.cpp

```diff
@@ -1044,9 +1057,9 @@ class ActiveExpiredStats final
 
     void Print(std::ostream& oss, const char* prefix, const char* name, const char* unit) const
     {
-        _all.Print(print(oss, prefix, "all", name), std::string_view(), unit);
-        _active.Print(print(oss, prefix, "active", name), std::string_view(), unit);
-        _expired.Print(print(oss, prefix, "expired", name), std::string_view(), unit);
+        _all.Print(oss, concat(prefix, name, "all"), unit);
+        _active.Print(oss, concat(prefix, "active", name), unit);
```

Comment:
What is this changing? Can you given an example string of before and after, please?

Reply:
Before:

document_active_views_all_counttotal 0
average 0
min 0
max 0

After:

document_active_views_all_count_total 1
document_active_views_all_count_average 1
document_active_views_all_count_min 1
document_active_views_all_count_max 1

I had that in my description, I separated the changes for the two commits.